nmt_chainer/eval.py

```diff
@@ -297,6 +302,8 @@ def define_parser(parser):
     
     parser.add_argument("--post_score_length_normalization", choices = ['none', 'simple', 'google'], default = 'simple')
     parser.add_argument("--length_normalization_strength", type = float, default = 0.2)
+    parser.add_argument("--post_score_coverage_penalty", choices = ['none', 'google'], default = 'none')
+    parser.add_argument("--coverage_penalty_strength", type = float, default = 0.2)
```

Comment:
you call this coverage_penalty_strength in the argument parsing, but you use post_score_coverage_penalty_strength in the rest of the code.